mozetl/landfill/sampler.py

```diff
@@ -56,6 +52,29 @@ def extract(sc, submission_date, sample=0.01):
     return landfill
 
 
+def _detect_telemetry_version(content_string):
+    """Detect document version from the payload itself.
+    Should match with the logic here:
+    https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/moz_telemetry/io_modules/decoders/moz_ingest/telemetry.lua#L162
+    If the given content string is not parseable as JSON,
+    default to a version of "0".
+    """
+    try:
+        content = json.loads(content_string)
+    except ValueError as e:
```

Comment:
```suggestion
    except ValueError:
```
This should probably fix the linting error.

Reply:
`json.loads` is throwing a `TypeError`.

```Caused by: org.apache.spark.api.python.PythonException: Traceback (most recent call last):
  File "/databricks/spark/python/pyspark/worker.py", line 262, in main
    process()
  File "/databricks/spark/python/pyspark/worker.py", line 257, in process
    serializer.dump_stream(func(split_index, iterator), outfile)
  File "/databricks/spark/python/pyspark/serializers.py", line 381, in dump_stream
    vs = list(itertools.islice(iterator, batch))
  File "/databricks/spark/python/pyspark/util.py", line 55, in wrapper
    return f(*args, **kwargs)
  File "/databricks/python/lib/python3.5/site-packages/mozetl/landfill/sampler.py", line 96, in _process
    doc_version = _detect_telemetry_version(content)
  File "/databricks/python/lib/python3.5/site-packages/mozetl/landfill/sampler.py", line 63, in _detect_telemetry_version
    content = json.loads(content_string)
  File "/usr/lib/python3.5/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'NoneType'```